js/views/modals/Settings/Settings.js

```diff
@@ -0,0 +1,88 @@
+import loadTemplate from '../../../utils/loadTemplate';
+import BaseModal from '../BaseModal';
+import SettingsGeneral from './SettingsGeneral';
+import SettingsPage from './SettingsPage';
+import $ from 'jquery';
+
+export default class extends BaseModal {
+  constructor(options = {}) {
+    const opts = {
+      removeOnClose: true,
+      modelContentClass: 'modalContent clrP border clrBr',
+      ...options,
+    };
+
+    super(opts);
+    this.options = opts;
+
+    this.tabViewCache = {};
+    this.tabViews = { SettingsGeneral, SettingsPage };
+
+    if (this.options.removeOnClose) this.on('close', () => this.remove());
+  }
+
+  className() {
+    return `${super.className()} settings`;
+  }
+
+  events() {
+    return {
+      'click .js-tab': 'tabClick',
+      'click .js-cancel': 'cancelClick',
+      'click .js-save': 'saveClick',
+      ...super.events(),
+    };
+  }
+
+  tabClick(e) {
+    const targ = $(e.target).closest('.js-tab');
+    this.selectTab(targ);
+  }
+
+  selectTab(targ) {
+    const tabViewName = targ.data('tab');
+    let tabView = this.tabViewCache[tabViewName];
+
+    if (!this.currentTabView || this.currentTabView !== tabView) {
+      this.$('.js-tab').removeClass('clrT active');
+      targ.addClass('clrT active');
+      if (this.currentTabView) this.currentTabView.$el.detach();
+      if (!tabView) {
+        tabView = new this.tabViews[tabViewName]();
+        tabView.render();
```

Comment:
You're never putting the tabView into the cache. Maybe after you create it you just need a `this.tabViewCache = tabView`.